src/tests/simulation/src/main.rs

```diff
@@ -109,6 +109,26 @@ async fn main() {
     println!("seed = {}", handle.seed());
     println!("{:?}", args);
 
+    // etcd node
+    handle
+        .create_node()
+        .name("etcd")
+        .ip("192.168.10.1".parse().unwrap())
+        .init(|| async {
+            let addr = "0.0.0.0:2388".parse().unwrap();
+            etcd_client::SimServer::builder()
+                // FIXME: If etcd returns timeout error, the meta will panic.
+                // To debug, change this rate to 0.1 and the sleep duration after meta to 30s.
+                // Tracking issue: https://github.com/risingwavelabs/risingwave/issues/4919
+                .timeout_rate(0.0)
```

Comment:
FYI, we've added the retry logic for etcd client in #5056. It should be okay to set a non-zero rate now. :)